Content.Server/Storage/Components/ArtifactStorageComponent.cs

```diff
@@ -1,14 +1,7 @@
-﻿using Content.Server.Xenoarchaeology.XenoArtifacts;
-
 namespace Content.Server.Storage.Components;
 
 [RegisterComponent]
-public sealed class ArtifactStorageComponent : EntityStorageComponent
+public sealed class ArtifactStorageComponent : Component
 {
-    [Dependency] private readonly IEntityManager _entMan = default!;
 
-    public override bool CanFit(EntityUid entity)
-    {
-        return _entMan.HasComponent<ArtifactComponent>(entity);
-    }
 }
```

Comment:
Couldn't you accomplish the same with a whitelist?

Reply:
You could, but since each artifact has it's own entity for each effect, it's cleaner to just check for the artifact component rather than having to manage a list.

Reply:
> it's cleaner to just check for the artifact component rather than having to manage a list.

I don't follow, you could just have a whitelist that checks for the component and it achieves the very same thing? Containers should have whitelist & blacklist support anyways, and then (afaik) this component would be redundant.